components/home/mcp-card.tsx

```
@@ -0,0 +1,185 @@
+import { vscDarkPlus, vs } from "react-syntax-highlighter/dist/esm/styles/prism"
+import { toast } from "sonner"
+import { getConfigJson } from "@/lib/utils"
+
+interface McpCardProps {
+    server: McpServerSpec
+    theme?: string
+    apiKey?: string
+}
+
+// Recursively inject envVars into localConfig where referenced by key
+function injectEnvVarsToLocalConfig(localConfig: any, envVars: any[] = [], apiKey: string | undefined) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)